src/main/java/com/sforce/ws/codegen/templates/simpleTypeOrganizationValue.st

```diff
@@ -0,0 +1,25 @@
+simpleTypeOrganizationValue(gen) ::= <<
+package $gen.packageName$;
+
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This is a generated class for the SObject Enterprise API.
+ * Do not edit this file, as your changes will be lost.
+ */
+public enum $gen.className$ {
+
```

Comment:
nit picks here:
-the comment text in the template does not really add much value so not sure why we need it at all.
-also personally would drop the xtra spaces